src/components/layout/ArPrice/ArPrice.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+
+import { useGlobalState } from '../../../state/contexts/GlobalState';
+import { byteSize } from '../../../utils/searchUtils';
+
+function ArPrice({ data }: { data: any }) {
```

Comment:
nit: any

Reply:
changed

Reply:
FWIW - it is better for the original commenter to resolve comments once they feel it's been addressed. i also don't see the change here.